apps/daimo-mobile/src/model/account.ts

```diff
@@ -150,6 +171,7 @@ export function parse(accountJSON?: string): Account | null {
       name: a.name,
       address: getAddress(a.address),
 
+      isDeployed: true, // TODO: is this okay to assume?
```

Comment:
this migration path will _probably_ be okay given current TestFlight set, but in future would be nice to be able to sequentially make migrations and have tests for different version storage